File: src/Classes/View/NavItems/NavItem.php
<?php
namespace MDJ\View\NavItems;

use MDJ\Interfaces\iView;
// use MDJ\src\Config;

class NavItem implements iView 
{
    protected string $page;
    protected string $name;
	protected string $li_class;
	protected string $a_class;

    public function __construct(string $page, string $name, string $li_class, string $a_class) 
	{
        $this->page = $page;
        $this->name = $name;
		$this->li_class = $li_class;
		$this->a_class = $a_class;
    }

    public function show() 
	{
        echo '<li class="'.$this->li_class.'"> 
                    <a class=" '.$this->a_class.'" href="'.\Config::BASEURL.'?page='.$this->page.'">'.$this->name.'</a>
              </li>';
    }
}

File: ArticleReviewApp/ArticleReviewApp/Models/NavItem.cs
using System;
using System.Collections.Generic;

namespace ArticleReviewApp.Models;

public partial class NavItem
{
    public string Page { get; set; } = null!;

    public string Name { get; set; } = null!;

    public string LiClass { get; set; } = null!;

    public string AClass { get; set; } = null!;

    public bool LoggedIn { get; set; }
}
